model/plugin_integration.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+const (
+	ScopeMobile = "mobile"
+	ScopeWebApp = "webapp"
+)
+
+type PluginIntegration struct {
+	PluginID   string      `json:"id"`
+	Location   string      `json:"location"`
```

Comment:
At a glance, it's not super clear how to interpret `Location` and `RequestURL` -- could we add GoDoc here?

Reply:
Sounds nice. Nevertheless, if you have any proposal for better naming, I am all in. I am not specially good at naming, and the naming of this whole infrastructure is still open to discussion.

Reply:
Final naming aside (and given that I'm only looking at this review), I currently can't tell you what these two fields do. Maybe even just write a doc using the current name, `Location define where the plugin should ...` and `RequestURL is used by the plugin to ...`.

Reply:
Addressed on 3f8023b